tests/abi/reference_binary.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file is distributed under the MIT License. See LICENSE.md for details.
+ */
+
+#include <setjmp.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "functions.h"
```

Comment:
Headers that have "side effects" (e.g., that you cannot include more than once, or define non-`static` functions) usually have the `.inc` extension.

Reply:
Sure.

Reply:
Done: https://github.com/revng/revng-qa/pull/4/commits/b375463f0ba22f1fa4cd2f8240d50300872e5d22.